sdks/python/src/opik/evaluation/models/litellm_chat_model.py

```diff
@@ -0,0 +1,68 @@
+import logging
+from typing import Any, Dict, List
+
+import litellm
+from litellm.types.utils import ModelResponse
+
+from . import base_model
+
+LOGGER = logging.getLogger(__name__)
+
+
+class LiteLLLMChatModel(base_model.OpikBaseModel):
+    def __init__(
+        self,
+        model_name: str = "gpt-3.5-turbo",
+        **kwargs: Any,
+    ) -> None:
+        super().__init__(model_name=model_name)
+
+        self._model_kwargs: Dict[str, Any] = kwargs
+        self._model_kwargs = self.cleanup_params(self._model_kwargs)
+
+        self._engine = litellm
+
+    def cleanup_params(self, params: Dict[str, Any]) -> Dict[str, Any]:
+        supported_params = set(
+            litellm.get_supported_openai_params(
+                model=self.model_name,
+            )
+        )
+
+        result = {}
+
+        for key in params:
+            if key not in supported_params:
+                LOGGER.warning(f"Unsupported parameter: '{key}' will be ignored.")
```

Comment:
I think it makes sense to raise an exception if parameter is not supported to avoid errors/unexpected errors on a later stage (generation).